### PR TITLE
Update SKILL.tsv

### PR DESCRIPTION
**Pull request by Zilox1994**
Edited lines 4800-5000
Not sure about:
4861: dont know if cryomancer or chronomancer.
4863: Not sure if it should be circle 1 pyromancer instead of Lv 1
4874: Not sure if A Handman's Knot. or Handman's Knot
4887: Not sure if it means to get tear gas at circle 3, or Lv3 tear gas
4907: Not sure if it means all stats or not.
4917: Not quite sure about this sentence
4918: Not quite sure about this sentence
4923: Not quite sure about this sentence

Didnt edit:
4807
4808
4810
4811
4812
4816
4818
4820
4821
4825
4826
4827
4831
4832
4836
4841
4844
4845
4846
4847
4849
4850
4851
4852
4904
4906
4914
4916
4920
4921
4922
4924
4925
4927
4928
4929
4933
4935
4936
4937
4939
4941
4942
4943
4945
4948
4949
4950
4955
4957
4958
4960
4962
4966
4967
4969
4970
4971
4972
4973
4974
4975
4976
4980
4981
4982
4984
4989
4994
4995
4996
4998
4999

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
...
-SKILL_20150717_004801	Cooldown time of [Dig] will be reduced by 10 secs.
-SKILL_20150717_004802	Physical ATK and Magic ATK will be increased for 30 mins for the allies that participated in [Item Awakening] dungeon. ATK will increase by 20% per Attribute level.
-SKILL_20150717_004803	Fulldraw : Knockback increase
-SKILL_20150717_004804	[Both hands bow] to the public type monster when mounted I will give a 20% additional damage of Physical attack.
-SKILL_20150717_004805	When [Swift Steps] is being activated, the enemies that got shot by [Oblique Shot] will fall into [Slow] with 5% probability per Attribute level for 7 secs.
-SKILL_20150717_004806	Physical DEF of the enemies that are threaded with [Full Draw] will decrease by 10% per Attribute level.
+SKILL_20150717_004800	Increases the range of [Disinter] by 10.
+SKILL_20150717_004801	Reduces the cooldown of [Dig] by 10 secs.
+SKILL_20150717_004802	Physical ATK and Magic ATK will be increased for 30 mins for allies that participated in the [Item Awakening] dungeon. ATK will increase by 20% per Attribute level.
+SKILL_20150717_004803	Fulldraw : Increases knockback
```
Comment (Sunaries) [inline SKILL.tsv:4803]: 'Knockback Increase' is fine. 'Increases knockback' sounds odd. Maybe 'Knockback Reinforce'?

File: SKILL.tsv [hunk 2]
```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
...
+SKILL_20150717_004841	HP recovery effect of [Restoration] increases per attribute level.
+SKILL_20150717_004842	Instant death probability of skill [Turn Undead] is affected by your spirit.
+SKILL_20150717_004843	Damage reflected off from [Iron Skin] will be added 20% of Physical ATK per Attribute level.
 SKILL_20150717_004844	An enemy that is shot by [Palm Strike] will fall into [Bleeding] for 8 secs. Duration of [Bleeding] will increase by 1 sec per Attribute level.
-SKILL_20150717_004845	[Pardoner] Make despeller
+SKILL_20150717_004845	[Pardoner] Make despeller.
 SKILL_20150717_004846	When you endow the effect that is registered at [Spell Shop] to your colleague, the duration will be increased. Duration will increase by 3 mins per Attribute level.
 SKILL_20150717_004847	Enemies nearby receive Holy damage when using [Arcane Energy]. Damage increases per attribute level.
-SKILL_20150717_004848	Duration of [Forecast] will increase by 1 sec per Attribute level.
+SKILL_20150717_004848	Increases duration of [Forecast] by 1 sec per Attribute level.
```
Comment (Sunaries) [inline SKILL.tsv]: Increases **the** duration

File: SKILL.tsv [hunk 3]
```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
...
+SKILL_20150724_004855	Requires Lv3 Warcry.
+SKILL_20150724_004856	Requires Lv2 Creating zombie.
+SKILL_20150724_004857	Requires Lv3 Creating zombie.
+SKILL_20150724_004858	Requires Lv3 Mackangdel.
+SKILL_20150724_004859	Requires Circle 2 Cataphract.
+SKILL_20150724_004860	Requires Circle 2 Cataphract.
+SKILL_20150724_004861	Requires Circle 2 Chronomancer.
+SKILL_20150724_004862	Requires Lv3 Deprotected zone.
+SKILL_20150724_004863	Requires Lv1 Pyromancer.
+SKILL_20150724_004864	Requires Cryomancer.
```
Comment (Sunaries) [inline SKILL.tsv]: Requires Cryomancer. -> Requires Circle 1 Cryomancer

File: SKILL.tsv [hunk 4]
```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
...
+SKILL_20150724_004854	Requires Lv 3 Kneeling Shot.
+SKILL_20150724_004855	Requires Lv3 Warcry.
+SKILL_20150724_004856	Requires Lv2 Creating zombie.
+SKILL_20150724_004857	Requires Lv3 Creating zombie.
+SKILL_20150724_004858	Requires Lv3 Mackangdel.
+SKILL_20150724_004859	Requires Circle 2 Cataphract.
+SKILL_20150724_004860	Requires Circle 2 Cataphract.
+SKILL_20150724_004861	Requires Circle 2 Chronomancer.
+SKILL_20150724_004862	Requires Lv3 Deprotected zone.
+SKILL_20150724_004863	Requires Lv1 Pyromancer.
```
Comment (Sunaries) [inline SKILL.tsv]: Requires Lv1 Pyromancer. -> Requires Circle 1 Pyromancer

File: SKILL.tsv [hunk 5]
```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
...
+SKILL_20150724_004865	Requires Lv3 Hovering.
+SKILL_20150724_004866	Requires Lv3 Ice wall.
+SKILL_20150724_004867	Requires Lv3 Catar Stroke.
+SKILL_20150724_004868	Requires Cross Guard.
+SKILL_20150724_004869	Requires Circle 2 Highlander.
+SKILL_20150724_004870	Requires Circle 3 Highlander.
+SKILL_20150724_004871	Requires Lv3 Finestra.
+SKILL_20150724_004872	Requires Circle 2 Hunter.
+SKILL_20150724_004873	Requires Lv3 Aukuras.
+SKILL_20150724_004874	Requires Lv3 Handman's Knot.
```
Comment (Sunaries) [inline SKILL.tsv]: Handman's Knot -> Hangman's Knot

File: SKILL.tsv [hunk 6]
```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
...
+SKILL_20150724_004898	Requires Lv3 Bash.
+SKILL_20150724_004899	Requires Lv3 Gung Ho.
+SKILL_20150724_004900	Requires Lv3 Lethargy.
+SKILL_20150724_004901	Requires Lv3 Reflect Shield.
+SKILL_20150724_004902	Requires Circle 2 Wizard.
+SKILL_20150724_004903	Requires Circle 3 Wizard.
+SKILL_20150729_004904	[Capri Sun] Physical/Hit/No Property.
+SKILL_20150729_004905	Increases HP and SP restoration.
+SKILL_20150729_004906	Restore HP +100{nl}Restore SP +100{nl}Duration -5 sec.
+SKILL_20150729_004907	Increases stats.
```
Comment (Sunaries) [inline SKILL.tsv]: _4907: Not sure if it means all stats or not._

Just stats.

File: SKILL.tsv [hunk 7]
```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
...
+SKILL_20150724_004878	Requires Lv2 Turn Undead.
+SKILL_20150724_004879	Requires Lv6 Indulgentia.
+SKILL_20150724_004880	Requires Circle 2 Pardoner.
+SKILL_20150724_004881	Requires Lv3 High Guard.
+SKILL_20150724_004882	Requires Lv6 Umbo Blow.
+SKILL_20150724_004883	Requires Lv6 Rim Blow.
+SKILL_20150724_004884	Requires Circle 2 Priest and Lv2 Resurrection.
+SKILL_20150724_004885	Requires Circle 2 Psychokino.
+SKILL_20150724_004886	Requires Circle 3 Psychokino.
+SKILL_20150724_004887	Requires Lv3 Tear Gas.
```
Comment (Sunaries) [inline SKILL.tsv]: _4887: Not sure if it means to get tear gas at circle 3, or Lv3 tear gas_

Lv. 3 is correct.

File: SKILL.tsv [hunk 8]
```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
...
-SKILL_20150729_004911	Skill cooldown time decreases
-SKILL_20150729_004912	Skill cooldown time decreases by 10%
-SKILL_20150729_004913	Physical ATK, Magic ATK increase
-SKILL_20150729_004914	Physical ATK +10%{nl}Magic ATK +10%
-SKILL_20150729_004915	Physical DEF, Magic DEF increase
-SKILL_20150729_004916	Physical DEF +10%{nl}Magic DEF +10%
-SKILL_20150729_004917	Open shop to repair weapon and temporarily increase its performance.
-SKILL_20150729_004918	Open shop to repair weapon and temporarily increase its performance. Repair is only for Armor, Pants, and Shield.
-SKILL_20150729_004919	Install basecamp for party members. Party members can use basecamp as storage or for teleport. Duration of buffs used around the camp will increase.
+SKILL_20150724_004853	Requires Lv1 Tincturing.
```
Comment (Sunaries) [inline SKILL.tsv]: This goes for line 4853 till 4903: remove the dot.

**Comment by Zilox1994** [comment]
Ok will do ^^. Why do some lines have dots and others dont :O?

**Comment by Sunaries** [comment]
@Zilox1994 None of the "Requires Circle/Lv. <skill>" have a dot at the end in the Korean file.

**Comment by Zilox1994** [comment]
Really appreciate your help man ^^

**Comment by imcgames** [comment]
:+1:

**Pull request merged**